keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -590,10 +592,12 @@ const (
 type ServiceImportSpec struct {
   // +listType=atomic
   Ports []ServicePort `json:"ports"`
-  // +kubebuilder:validation:MaxItems:=1
+  // +kubebuilder:validation:MaxItems:=2
```

Comment:
Shouldn’t we have this on `IPFamilies` too?

Reply:
Oops! Just read the diff wrong.